programs/range/src/main.rs

```diff
@@ -278,43 +283,8 @@ where
             Ok(header) => header,
             Err(e) => {
                 error!(target: "client", "Failed to execute L2 block: {}", e);
-
-                if cfg.is_holocene_active(attributes.payload_attributes.timestamp) {
-                    println!("cycle-tracker-report-start: block-execution");
-                    // Retry with a deposit-only block.
-                    warn!(target: "client", "Flushing current channel and retrying deposit only block");
-
-                    // Flush the current batch and channel - if a block was replaced with a
-                    // deposit-only block due to execution failure, the
-                    // batch and channel it is contained in is forwards
-                    // invalidated.
-                    pipeline.signal(Signal::FlushChannel).await?;
-
-                    // Strip out all transactions that are not deposits.
-                    attributes.transactions = attributes.transactions.map(|txs| {
-                        txs.into_iter()
-                            .filter(|tx| (!tx.is_empty() && tx[0] == OpTxType::Deposit as u8))
-                            .collect::<Vec<_>>()
-                    });
-
-                    // Retry the execution.
-                    block_executor = executor.new_executor(cursor.l2_safe_head_header().clone());
-                    let res = block_executor.execute_payload(attributes.clone());
-                    println!("cycle-tracker-report-end: block-execution");
-                    match res {
-                        Ok(header) => header,
-                        Err(e) => {
-                            error!(
-                                target: "client",
-                                "Critical - Failed to execute deposit-only block: {e}",
-                            );
-                            return Err(DriverError::Executor(e));
-                        }
-                    }
-                } else {
-                    // Pre-Holocene, discard the block if execution fails.
-                    continue;
-                }
```

Comment:
You shouldn't need to modify this logic, as it already has the check for whether holocene is active. Would recommend keeping it to reduce the diff.

Reply:
The cfg is part of RollupConfig, which is defined in op-alloy. Since we removed the Holocene logic from op-alloy, we also removed it here for consistency.